pkg/operator/latencyprofilecontroller/controller_test.go

```diff
@@ -0,0 +1,142 @@
+package latencyprofilecontroller
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	controlplanev1 "github.com/openshift/api/kubecontrolplane/v1"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+)
+
+func TestConfigMatchesControllerManagerArgument(t *testing.T) {
+	createConfigMapFromCMConfig := func(
+		config controlplanev1.KubeControllerManagerConfig,
+		configMapName, configMapNamespace string,
+	) (configMap corev1.ConfigMap) {
+
+		configAsJsonBytes, err := json.MarshalIndent(config, "", "")
+		require.NoError(t, err)
+
+		configMap = corev1.ConfigMap{
+			ObjectMeta: v1.ObjectMeta{Name: configMapName, Namespace: configMapNamespace},
+			Data: map[string]string{
+				controllerManagerConfigMapKey: string(configAsJsonBytes),
+			},
+		}
+		return configMap
+	}
+
+	cmConfigs := []controlplanev1.KubeControllerManagerConfig{
+		// config 1
+		{
+			ExtendedArguments: map[string]controlplanev1.Arguments{},
```

Comment:
nit: we don't have to initialize the map here

Reply:
Done